streams/src/main/java/org/apache/kafka/streams/TopologyDescription.java

```diff
@@ -57,7 +57,7 @@
     /**
      * Represents a {@link Topology#addGlobalStore(org.apache.kafka.streams.processor.StateStoreSupplier, String,
      * org.apache.kafka.common.serialization.Deserializer, org.apache.kafka.common.serialization.Deserializer, String,
-     * String, ProcessorSupplier)} global store}.
+     * String, org.apache.kafka.streams.processor.ProcessorSupplier)} global store}.
```

Comment:
Yes, since the `ProcessorSupplier` is in different package, I have provided the fully qualified name to resolve the warning in intellij.